well-sensor-esp32.ino

```diff
@@ -41,6 +41,16 @@ void setup() {
   initQr();
  
   if (initSPIFSS()) {
+    
+    //set configs for INA with addr = (A1 = GND & A2 = GND)
+    setPowerMeterConfigs(powerMeterConfigs[0], 0b10000000, INA260::MODE_ISH_VBUS_CONTINUOUS, INA260::ISHCT_1_1MS,
```

Comment:
Looks like you set wrong have address you set 0b10000000 which is 0x80? but in the spec you have 1000000 which is 0x40. The same with 2nd address. Please, check 
![image](https://user-images.githubusercontent.com/37577752/61791738-82058900-ae23-11e9-8230-76ee3973a15f.png)